Frameworks/UIKit/UITextView.mm

```diff
@@ -759,7 +763,7 @@ - (CGSize)sizeThatFits:(CGSize)fitSize {
     centerRect.origin.y = 0;
     centerRect.size = fontExtent;
     // TODO(DH)
-    //EbrCenterTextInRectVertically(&centerRect, &fontExtent, _font);
+    // EbrCenterTextInRectVertically(&centerRect, &fontExtent, _font);
```

Comment:
remove commented code?  also looks like there is a missing TODO issue, lets create it now.